pyanaconda/ui/gui/__init__.py

```diff
@@ -731,38 +730,6 @@ def _widgetScale(self):
             # Export the scale so that Gtk programs launched by anaconda are also scaled
             iutil.setenv("GDK_SCALE", "2")
 
-    def _convertSignals(self):
```

Comment:
Are we sure we want to remove this ? It looks like something that should be in use, unless it was superseded by something else.